### DISCUSSION TITLE
직사각형 뷰도 테스트 한번 해보는 건 어떨까요?

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#25] [제보하기]
직사각형이라고 하니 의미가 모호한데. 
현대 쿼터뷰는 45도 정도 기울어져 있는데요, 기울임 없이 상하 좌우 격자로 구성된 뷰는 어떨까요?

이동할때 주고 상하좌우 또는 WASD 키로 이동하는데  기울어져 있으니까 이동하는데 뭔가 많이 눌러줘야 되는 느낌이라. 
게더 같이 기울임이 없는 구성일 때 사용성은 어떨까 싶어서 의견 올려봅니다.

**Comment by ACT-TJ** [comment]
개선 관련 너무 좋은 의견 감사드립니다.^^

각도에 대해 많은 고민을 하고 쿼터뷰를 선택을 하였으나, 
실제 유저분들의 의견과 편의는 지속적으로 확인을 하며 수정해 가야할 것 같습니다.

현재 마우스 우클릭을 통해 맵 전체를 회전해 보실 수 있도록 기능을 제공하고 있는데요,
캐릭터 컨트롤의 방향은 고정을 시켜놓은 관계로 회전값은 클릭을 풀면 다시 원상복구 됩니다. 

이 부분을 복구되지 않고 유지하며 유저가 원하는 각도로 플레이 될 수 있도록 개선 예정입니다.
해당 부분 개선 후 사용자가 편한 각도로 맞춰 놓고 사용할 수 있게 되어 말씀하신 부분도
커버가 되고, 유저들이 선호하는 각도도 데이터 수집을 통해 좀 더 구체화 할 수 있을 것으로 것으로 
기대하고 있습니다.^^  

* 캐릭터의 이동 관련한 부분으로 전체 점검이 필요하여 Alpha 기간 내에는 어려울 수 있으나, ACT 오픈 전에는
  변경될 수 있도록 해당 개선 건 잘 Follow 하도록 하겠습니다

발전하는 모습 기대해주세요!! 감사합니다!
